samples/03_org_administrators/clone_storymap_version2.ipynb

```diff
@@ -0,0 +1,709 @@
+{
```

Comment:
<p>"infromation in the Classic Story Map."  -&gt; typo in information. And, did you mean ArcGIS StoryMap instead of classic?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/797/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>I removed this sentence entirely. it's redundant given I explain the differences in how to obtain resources from ArcGIS StoryMaps versus Classic Story Maps in the <strong>Get the </strong><code><strong>ArcGIS StoryMap</strong></code><strong>  and export its resources </strong>section.</p>